README.md

```diff
@@ -31,6 +31,30 @@ apply plugin: "com.dipien.bye-bye-dead-code"
 
 ### Tasks
 
+#### Generate unused code report 
+
+This task depends on R8 which detects unused code when you build your project. So you must enable `minifyEnabled` in the `build.gradle` of your app module. [See how to configure R8](https://developer.android.com/studio/build/shrink-code)
+
+```
+./gradlew app:bundleRelease generateDeadCodeReport
+```
+
+The report `app/build/outputs/dead_code/usage.txt` will be generated.
+
+##### Extension properties
+
+The extension is configured by default to work using the release build type, but you can change this if you want. Using the extension `byeByeDeadCode` you can modify the following properties:
+
+Name | Type | Description
+--- | --- | ---
+ignoredClasses | List | *Optional.* List of ignored classes. Supports regex.
+ignoredMembers | List | *Optional.* List of ignored class members. Supports regex.
+generatedClassesDirs | List | *Required.* Directories that are used by different tools (e.g. kapt) to store generated source code. <br/><br/> Default: <br/>["build/generated/source/kapt/release", <br/>"build/generated/source/navigation-args/release"]
+compiledKotlinClassesDir | String | *Required.* Directory where compiled kotlin classes are stored. <br/><br/> Default: "build/tmp/kotlin-classes/release"
+compiledJavaClassesDir | String | *Required.* Directory where compiled Java classes are stored. <br/><br/> Default: "build/intermediates/javac/release"
+proguardUsageFilePath | String | *Required.* File path of usage.txt generated by R8. <br/><br/> Default: "app/build/outputs/mapping/release/usage.txt"
```

Comment:
proguard or r8?

Reply:
I renamed proguardUsageFilePath to r8UsageFilePath